2/README.md

```diff
@@ -54,29 +54,104 @@ There are X main use cases for Template:
 
 A Tango Device is a strict definition of a distributed object. 
 * configuration are represented in the form of properties.
-* data are represented in the form of attributes.
+* data are represented in the form of attributes and pipes.
 * actions are represented in the form of commands.
 
-Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, Attribute, Command following the rules below:
-
+Its model can be represented as a defined tree which each elements are from a defined types: Class, Device, Property, 
+Attribute, Pipe, Command following the rules below:
 
 * The Device is a distributed object which SHALL be accessed locally or via network.
-* The Device SHALL be an instance of one Class
-* The Device MAY have one or several Property, called Device Property
-* The Device MAY have one or several Attribute
+* The Device SHALL be an instance of one Device Class, see [RFC-9]()
+* The Device MAY have one or several Property, called Device Property, see [RFC-5]()
+* The Device MAY have one or several Attribute, see [RFC-4]() 
+* The Device MAY have one or several Pipe
+* The Device MAY have one or several Command, see [RFC-3]()
+
+
 * The Device SHALL have one State attribute 
 * The Device SHALL have one Status attribute
-* The Device MAY have one or several Command
-* The Device SHALL have one unique identifier which represents its Name
+* The Device SHALL have one State command
+* The Device SHALL have one Status command
+
+**Note**: Future specification may remove 'SHALL have' requirements for State and Status commands.
+
+
+[TO-DO: Should the two points below be moved to Class RFC?]
+* The Device Class MAY have one or more Device instance
+* The Device Class MAY have one or several Property, called Class Property
+
```

Comment:
@hardion, the points above seem to belong to Device Class (RFC-9). Since there is no RFC-9 yet, should I create one and move these there, so it will not be forgotten?